src/native/libs/System.Native/entrypoints.c

```diff
@@ -48,6 +48,9 @@ static const Entry s_sysNative[] =
     DllImportEntry(SystemNative_GetSignalForBreak)
     DllImportEntry(SystemNative_SetSignalForBreak)
     DllImportEntry(SystemNative_GetSystemTimeAsTicks)
+#if defined(TARGET_WASI) || defined(TARGET_BROWSER)
```

Comment:
I don't know why `SystemNative_GetDefaultTimeZone` for Android doesn't need to be here too ?

@steveisok any tips ?